include/dlaf/communication/broadcast_panel.h

```diff
@@ -57,7 +41,7 @@ std::pair<SizeType, comm::IndexT_MPI> transposedOwner(const matrix::Distribution
 /// @param serial_comm  where to pipeline the tasks for communications.
 /// @pre Communicator in @p serial_comm must be orthogonal to panel axis
 template <class T, Device D, Coord axis, matrix::StoreTransposed storage,
-          class = std::enable_if_t<!std::is_const_v<T>>>
+          std::enable_if_t<!std::is_const_v<T>, int> = 0>
```

Comment:
Just out of curiosity: any particular reason for this change?

Reply:
it's completely unrelated to the topic of the pr, he took the chance to make this templates less sensitive to potential problems (by passing a type, the previous one could have been tricked to "skip" the sfinae check)